lab-steven/routes/auth-routes.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const debug = require('debug')('cfgram: authRoutes');
+const basicAuth = require('../lib/basic-auth-middleware');
+const authCtrlr = require('../controller/auth-controller');
+
+module.exports = function(router){
+  router.post('/signup', (req, res) => {
+    debug('#POST signup');
+
+    let tempPass = req.body.password;
```

Comment:
this will be handled in your controller, not in the route per scott's instruction. it looks like solid logic though.